src/prometheus/http.zig

```diff
@@ -41,6 +41,7 @@ pub fn getMetrics(
     _: *httpz.Request,
     response: *httpz.Response,
 ) !void {
+    response.content_type = .TEXT; // expected by prometheus
```

Comment:
what happens when you don't include this?

Reply:
Prometheus doesn't record it at all.
```
prometheus     | time=2025-01-06T15:04:39.954Z level=ERROR source=scrape.go:1590 msg="Failed to determine correct type of scrape target." component="scrape manager" scrape_pool=prometheus target=http://host.docker.internal:12345/metrics content_type="" fallback_media_type="" err="non-compliant scrape target sending blank Content-Type and no fallback_scrape_protocol specified for target"
```

Reply:
I have no idea - worth noting that our prometheus image doesn't have a version on it, pretty sure that means we're tracking the latest image, which is probably undesirable.

Reply:
added a version - 01654a9f